lib/javascript/StartupHelpers.js

```diff
@@ -22,12 +23,28 @@ const { getAllChunks } = require("./ChunkHelpers");
 /** @typedef {import("../RuntimeTemplate")} RuntimeTemplate */
 /** @typedef {(string|number)[]} EntryItem */
 
+const getModuleFederationPlugin = memoize(() =>
+	require("../container/ModuleFederationPlugin")
+);
```

Comment:
I don't think this is the right place for such imports. And yes looks like we need an another place for hook

Reply:
Totally agree. This work primarily illustrates what we must do, where we need more flexibility. Eagerly appreciate some improvement here for sure